setup.py

```diff
@@ -4,10 +4,13 @@
     long_description = fh.read()
 
 setuptools.setup(
-    name="great_expectations_airflow",
+    name="airflow-provider-great-expectations",
     version="0.0.1",
-    description="An Airflow operator for Great Expectations",
+    description="An Apache Airflow provider for Great Expectations",
+    long_description=long_description,
+    long_description_content_type='text/markdown',
     url="https://github.com/great-expectations/great_expectations_airflow",
     packages=setuptools.find_packages(),
     python_requires='>=3.6',
+    package_data={'': ['./great_expectations_provider/examples/data/*.csv']},
```

Comment:
@ashb is this what you're thinking re: including data files in the dist?

Reply:
Something like this, maybe yes.

If you run `python setup.py bdist_wheel && unzip -l dist/*.whl` you should see the .csv files in the right place. Do you?

Reply:
Fixed it up- I see 'em now :).